nima/fault-tolerance/src/test/java/io/helidon/nima/faulttolerance/BulkheadTest.java

```diff
@@ -42,14 +44,19 @@ class BulkheadTest {
 
     private static final long WAIT_TIMEOUT_MILLIS = 10000;
 
-    private final CountDownLatch enqueuedSubmitted = new CountDownLatch(1);
+    private CountDownLatch enqueuedSubmitted;
```

Comment:
This change is done to allow `@RepeatedTest` to be used on `testBulkhead` to reproduce intermittent failure